phpstan-baseline.neon

```diff
@@ -1,5 +1,35 @@
 parameters:
 	ignoreErrors:
+		-
+			message: "#^Call to an undefined method RedisCluster\\:\\:unlink\\(\\)\\.$#"
+			count: 1
+			path: src/Cache/Engine/RedisEngine.php
+
+		-
+			message: "#^Call to an undefined method Redis\\|RedisCluster\\:\\:connect\\(\\)\\.$#"
+			count: 2
+			path: src/Cache/Engine/RedisEngine.php
+
+		-
+			message: "#^Call to an undefined method Redis\\|RedisCluster\\:\\:pconnect\\(\\)\\.$#"
+			count: 2
+			path: src/Cache/Engine/RedisEngine.php
+
+		-
+			message: "#^Call to an undefined method Redis\\|RedisCluster\\:\\:unlink\\(\\)\\.$#"
+			count: 1
+			path: src/Cache/Engine/RedisEngine.php
+
+		-
+			message: "#^Method RedisCluster\\:\\:scan\\(\\) invoked with 4 parameters, 1\\-3 required\\.$#"
+			count: 2
+			path: src/Cache/Engine/RedisEngine.php
+
+		-
+			message: "#^Parameter \\#3 \\$pattern of method RedisCluster\\:\\:scan\\(\\) expects int, string given\\.$#"
```

Comment:
IMO this should be a cast instead.

Reply:
@dereuromark I think this is some kind of signature mismatch on phpstan.
For `Redis Cluster` the scan call goes like this
```php
scan(int|null $iterator, string|array $key_or_address, string|null $pattern = null, int $count = 0)
```

and for Redis its
```php
scan(int|null $iterator, string|null $pattern = null, int $count = 0, string $type = NULL)
```

The errors happens on lines that use `Redis Cluster scan`